synfig-studio/src/gui/app.cpp

```diff
@@ -1349,19 +1362,22 @@ DEFINE_ACTION("keyframe-properties", _("Properties"));
 	ACCEL2(Gtk::AccelKey(')',Gdk::CONTROL_MASK,					"<Actions>/canvasview/increase-low-res-pixel-size"			));
 	ACCEL2(Gtk::AccelKey('(',Gdk::MOD1_MASK|Gdk::CONTROL_MASK,			"<Actions>/action_group_layer_action_manager/amount-dec"		));
 	ACCEL2(Gtk::AccelKey(')',Gdk::MOD1_MASK|Gdk::CONTROL_MASK,			"<Actions>/action_group_layer_action_manager/amount-inc"		));
-	ACCEL2(Gtk::AccelKey(']',Gdk::CONTROL_MASK,					"<Actions>/canvasview/jump-next-keyframe"				));
-	ACCEL2(Gtk::AccelKey('[',Gdk::CONTROL_MASK,					"<Actions>/canvasview/jump-prev-keyframe"				));
 	ACCEL2(Gtk::AccelKey('=',Gdk::CONTROL_MASK,					"<Actions>/canvasview/canvas-zoom-in"					));
 	ACCEL2(Gtk::AccelKey('-',Gdk::CONTROL_MASK,					"<Actions>/canvasview/canvas-zoom-out"					));
 	ACCEL2(Gtk::AccelKey('+',Gdk::CONTROL_MASK,					"<Actions>/canvasview/time-zoom-in"					));
 	ACCEL2(Gtk::AccelKey('_',Gdk::CONTROL_MASK,					"<Actions>/canvasview/time-zoom-out"					));
-	ACCEL2(Gtk::AccelKey('.',Gdk::CONTROL_MASK,					"<Actions>/canvasview/seek-next-frame"					));
-	ACCEL2(Gtk::AccelKey(',',Gdk::CONTROL_MASK,					"<Actions>/canvasview/seek-prev-frame"					));
-	ACCEL2(Gtk::AccelKey('>',Gdk::CONTROL_MASK,					"<Actions>/canvasview/seek-next-second"					));
-	ACCEL2(Gtk::AccelKey('<',Gdk::CONTROL_MASK,					"<Actions>/canvasview/seek-prev-second"					));
+	ACCEL("bracketleft",							"<Actions>/canvasview/jump-prev-keyframe");
+	ACCEL("bracketright",							"<Actions>/canvasview/jump-next-keyframe");
+	ACCEL("comma",									"<Actions>/canvasview/seek-prev-frame");
+	ACCEL("period",									"<Actions>/canvasview/seek-next-frame");
+	ACCEL("<Shift>less",							"<Actions>/canvasview/seek-prev-second");
+	ACCEL("<Shift>greater",							"<Actions>/canvasview/seek-next-second");
+	ACCEL("<Control><Shift>less",					"<Actions>/canvasview/seek-begin");
+	ACCEL("<Control><Shift>greater",				"<Actions>/canvasview/seek-end");
```

Comment:
Now I realized vertical alignment using tabs is a bad idea
The above is 8 width -> inconsistent
Change it to 4 width -> consistent

I suggest using spaces for vertical alignment to force width consistency.

Below are two short experiments I did for demonstration:
[tabs alignment.zip](https://github.com/synfig/synfig/files/5382619/tabs.alignment.zip)
[spaces alignment.zip](https://github.com/synfig/synfig/files/5382620/spaces.alignment.zip)

Reply:
Yes, it should be spaces. Tabs for indentation level only.